client/src/components/ProductCard.tsx

```diff
@@ -6,7 +6,13 @@ import { UserLogs } from '../context/LogContext';
 import BkmarkButton from './BkmarkButton';
 import LikeButton from './LikeButton';
 import LogButton from './LogButton';
-import './ProductCard.css';
+
+import '../styles/Card.css';
+import '../styles/Container.css';
+import '../styles/Fonts.css';
+import '../styles/Icons.css';
+import '../styles/Button.css';
+import '../styles/Images.css';
```

Comment:
Please remove these extra imports and revert to a single `ProductCard.css` file.